azure/batch/scripts/train_model_finetune_on_catalog.py

```diff
@@ -38,6 +39,15 @@
     parser.add_argument('--debug', dest='debug', default=False, action='store_true')
     args = parser.parse_args()
 
+    schema_dict = {
+        'cosmic_dawn': cosmic_dawn_ortho_schema,
+        'euclid': {
```

Comment:
We agreed to have hard-coded for test now as we aren't on the latest zoobot version - We plan to upgrade but this just for test purposes. This would be removed immediately after

Reply:
Can you add an issue so we don't forget to do this?

Reply:
Yes, added that here: https://github.com/zooniverse/bajor/issues/55